RunScripts/en-de_pipeline.sh

```diff
@@ -1,10 +1,11 @@
 #!/bin/bash
 source activate unmt
 
-OUTPUT_DIR=$1
+EX_ABBR=clipL_sFz_transf
```

Comment:
Should this these changes be forked in a `RunScripts/en-de_beit_pipeline.sh` to keep the existing I2I unaffected?

Reply:
Good point, I am like 98% sure this won't influence anything since that only serves as a general argument to tell the script to save the model to a different folder. I can name it `en-de_pipeline_abblation.sh` so that we don't conflict!